tools/st2_deploy.sh

```diff
@@ -266,11 +266,11 @@ fi
 touch $config
 cat <<mistral_config >$config
 [database]
-connection=mysql://mistral:StackStorm@localhost/mistral
-max_pool_size=50
+connection = mysql://mistral:StackStorm@localhost/mistral
+max_pool_size = 100
```

Comment:
Please double-check with @lakshmi-kannan and @m4dcoder they did some experiments on these numbers.